bin/cli.js

```diff
@@ -202,12 +202,23 @@ new Promise((resolve, reject) => {
 				exitCode = 1;
 			}
 
+			if (program.image) {
+				return res.circularImage(program.image).then((imagePath) => {
+					spinner.succeed(
+						`${chalk.bold('Image created at')} ${chalk.cyan.bold(imagePath)}`
+					);
+					return res;
+				});
+			}
+
```

Comment:
Delete this addition.
```suggestion
```
These lines cause the build to fail with the following linting error,
```
error  Arrow function has a complexity of 11                               complexity
error  Arrow function has too many statements (26). Maximum allowed is 25  max-statements
```
I have a solution for this.
You can add a check for it in the ```image``` method of ```lib/api.js``` at line 151 ,
```
if (this.circular) {
    return this.circularImage(imagePath);
} else {
    return graph.image(this.obj(), this.circular(), imagePath, this.config);
}
```